index.Rmd

```diff
@@ -276,7 +322,9 @@ knitr::include_graphics("images/transform.png")
 
 ---
 
-You may not want to use the default stat... you can override the defaults by specifying a stat. Stats can be called using `stat_` or the shorthand `y=..stat..`
+You may not want to use the default stat... you can override the defaults by specifying a stat. Stats can be called as we have already seen using `stat_` to specify the statistical transformation.
+
+In cases where we just have an x variable (e.g. for a simple bar chart), we can also just specify the stat we want as a y co-ordinate mapping. For example, in the aes mapping we simply call `y = stat(count)` for a basic count of the x variable, `y = stat(mean)` for the mean, `y = stat(prop)` for the proportion of observations in each x category, etc.
 
 ```{r, fig.height=2}
 #Proportion chart
```

Comment:
probably want to replace ..prop.. with stat(prop) to align the explanations. That said, worth noting that the stat() function is only available in ggplot2 3.0.0 onwards (not everyone will have that by default), so might be worth retaining a reference to the `..stat..` form, referring to it as an older alternative.

Reply:
Thanks Thomas, good point, I'd missed that the example was still using ..prop.. . 

I've added some narrative on it and changed the example as suggested.